src/api/explorer.js

```diff
@@ -153,27 +207,78 @@ export const getHistoryTransfers = async (substrate_address) => {
       filterAssetTransfers: filterVariable,
       filterStakings: stakingsFilter,
       orderByStakings: orderByVariable,
+      filterSoraMinted,
+      filterSoraBurned,
     },
   });
-  const assetsData = await getAdditionalAssets(substrate_address, true);
-  const stakingsData = result.data?.data?.stakings?.nodes;
-  const transfersAssets = result.data?.data?.assetTransfers?.nodes;
+  return data;
+};
+
+const parseSoraTransfer = (soraMinted, soraBurned, assetsData) => {
+  const soraBridgeName = 'Sora Bridge';
+  const soraMintedParsed = soraMinted.map(({
+    asset, block, recipientId, value,
+  }) => {
+    const parsedAsset = JSON.parse(asset);
+    const isLLD = Object.keys(parsedAsset)[0] === 'lld' && parsedAsset.lld === null;
+
+    return {
+      asset: isLLD ? 'LLD' : assetsData[parsedAsset.asset].metadata.symbol,
+      decimals: isLLD ? null : assetsData[parsedAsset.asset].metadata.decimals,
+      fromId: soraBridgeName,
```

Comment:
Technically we have data on the exact Sora Network account that is the sender (for mints) and the recipient (for burns). I think just putting `Sora Bridge` is fine, just leaving this as comment for Dorian in case he wants it different.